### ISSUE TITLE
Regression with v3.2.1+

### ISSUE THREAD

**Issue by dlong500** [leandrowd/react-responsive-carousel#430]
After updating from v3.1.57 to v3.2.1 or above of react-responsive-carousel my project fails to compile throwing the following:
```
C:\local\code\repo\node_modules\react-responsive-carousel\lib\cjs\shims\document.js:7
var _default = document;
               ^
ReferenceError: document is not defined
    at Object.<anonymous> (C:\local\code\repo\node_modules\?[4mreact-responsive-carousel?[24m\lib\cjs\shims\document.js:7:16)
```
I'm not seeing any notice of breaking changes or anything else that might explain it.  Reverting to v3.1.57 restores working behavior.

**Comment by leandrowd** [comment]
Hey @dlong500 , can you provide more details about your system and how you got the error? Was it at compile time? Are you using any bundler? Thanks

**Comment by guramidev** [comment]
@leandrowd This is happening for me as well. We are using react-responsive-carousel in our app which uses SSR

**Comment by dlong500** [comment]
I compile my project with webpack and babel, and this occurs during one of those phases.  I don't have a lot of extra info yet at this time because I had to immediately revert back to a working version.  I can provide more info if necessary when I can get a bit of time.

**Comment by dlong500** [comment]
I did a bit more testing, and it actually is not during compilation but rather during runtime that it fails (on the server side).  It appears to cause the Node process to reference the document object (which of course isn't valid for server-side).  This seems directly related to the shim added recently.

**Comment by leandrowd** [comment]
New version is published which should have this error resolved. Please let me know how the update goes. Thanks

**Comment by dlong500** [comment]
Thanks!  The issue is resolved for me with v3.2.4.
